docs/UserGuide.md

```diff
@@ -148,6 +149,19 @@ Examples:
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
 
+### Sorting the list of clients : `sort`
+
+Sort the list of clients in ClientBook by their names in alphabetical order.
+
+Format: `sort DIRECTION`
+
+* Sort the list of clients according the specified `DIRECTION`.
```

Comment:
Is using "sorts" instead of "sort" better as it follows the convention used by the other commands?

Reply:
Noted thanks for the review